NGitLab/Models/AccessLevel.cs

```diff
@@ -19,5 +19,6 @@ public enum AccessLevel
         /// Only valid for groups.
         /// </summary>
         Owner = 50,
+        NoAccess = 0,
```

Comment:
You should move it to the first position

Reply:
[According to the documentation](https://docs.gitlab.com/ee/api/protected_branches.html) there is an `Admin = 60` mode